src/main/scala/edu/berkeley/ce/sparkrocks/Block.scala

```diff
@@ -476,7 +474,7 @@ case class Block(center: Array[Double], faces: Seq[Face], generation: Int=0) ext
         linProg.addConstraint(faceCoeffs, LinearProgram.LE, rhs)
       }
       val s = linProg.solve().get._2
-      math.abs(s - face.d) <= NumericUtils.EPSILON
+      math.abs(s - face.d) < NumericUtils.EPSILON / 10.0
```

Comment:
If we're not going to be using `NumericUtils.EPSILON` directly anymore, I would suggest we make this another constant and include it in the `Block` object. That way, if we ever change `NumericUtils.EPSILON`, this will stay the same, and it also seems more clear.

How does that sound to you?